solr/solrj/src/java/org/apache/solr/common/params/CommonParams.java

```diff
@@ -160,6 +160,26 @@
    */
   String TIME_ALLOWED = "timeAllowed";
 
+  /**
+   * Is the query cancellable?
+   */
+  String IS_QUERY_CANCELLABLE = "canCancel";
+
+  /**
+   * Custom query UUID if provided.
+   */
+  String CUSTOM_QUERY_UUID = "queryUUID";
+
+  /**
+   * UUID for query to be cancelled
+   */
+  String QUERY_CANCELLATION_UUID = "cancelUUID";
+
+  /**
+   * UUID of the task whose status is to be checked
+   */
+  String TASK_CHECK_UUID = "taskUUID";
```

Comment:
We may leave this as is (although I would be tempted to use `queryUUID` here too) if you think we will have other types of cancellable tasks in the future.

Reply:
We will (I plan to get to the long running collection creation soon)